ci_framework/roles/edpm_build_images/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install required packages
+  become: true
+  ansible.builtin.package:
+    name: "{{ cifmw_build_host_packages }}"
+    state: latest
+
```

Comment:
@bshewale 

edpm-builder-image rpm was recently added.


For periodic jobs, we want to install edpm-builder-image from rpm.

For check jobs we want to use the zuul cloned repo and not install edpm-builder-image rpm (for now till we add support in ci-framework to build rpm using dlrn from github repos)

Please add a task here to install edpm-builder-image based on some var which we will toggle in check jobs.

Reply:
fyi.. path to rpm
https://trunk.rdoproject.org/centos9-antelope/component/podified/podified-ci-testing/edpm-image-builder-0.0.1-0.20230606143847.a2315ff.el9.noarch.rpm

Reply:
added support to install from rpm